src/sb_pointcloud_processing/launch/line_extractor_rviz_degree_3.launch

```diff
@@ -14,6 +14,7 @@
         <param name="x_delta" value="0.1" type="double" />
         <!-- size of marker points for rviz -->
         <param name="scale" value="0.1" type="double" />
+        <remap from="input_pointcloud" to="/height_filter/output" />
```

Comment:
there's no need to change this file since this is a launch file specifically for testing rviz, and does not affect the overall pipeline.

Reply:
👌